src/main/java/ava/processor/TaskList.java

```diff
@@ -0,0 +1,93 @@
+package ava.processor;
+
+import java.util.ArrayList;
+
+import ava.task.Task;
+
+/**
+ * Class to represent the tasks lists.
+ */
+public class TaskList {
+    private static ArrayList<Task> tasks;
+
+    /**
+     * The constructor for TaskList with no parameters.
+     */
+    public TaskList() {
+        this.tasks = new ArrayList<Task>();
+    }
+
+    /**
+     * The constructor for TaskList with an input of ArrayList.
+     *
+     * @param task An ArrayList of tasks.
+     */
+    public TaskList(ArrayList<Task> task) {
+        this.tasks = task;
+    }
+
+    /**
+     * Adds a specific tasks to the TaskList.
+     *
+     * @param t A specific tasks to add.
```

Comment:
"tasks" might be better as singular "task" here, since we are dealing with one task. I noticed this in the other functions as well. From my understanding, it could have arose from a refactor of "task" -> "tasks".